docs/overview.md

```diff
@@ -13,22 +16,48 @@ As a versatile solution, ScalarDB supports a range of databases, including:
 
 For details on which databases ScalarDB supports, refer to [Supported Databases](scalardb-supported-databases.md).
 
+## Why ScalarDB?
+
+Several solutions, such as global transaction managers, data federation engines, and HTAP systems, have similar goals, but they are limited in the following perspectives:
+
+- Global transaction managers (like Oracle MicroTx and Atomikos) are designed to run transactions across a limited set of heterogeneous databases (like only XA-compliant databases).
+- Data federation engines (like Denodo and Starburst) are designed to run analytical queries across heterogeneous databases.
+- HTAP systems (like TiDB and SingleStore) run both transactions and analytical queries only on homogeneous databases.
+
+In other words, they virtually unify databases in an incomplete way. For example, with data federation engines, users can run read-only analytical queries on a virtualized view across multiple databases. However, they often need to run update queries separately for each database.
+Unlike other solutions, ScalarDB stands out by offering the ability to run both transactional and analytical queries on heterogeneous databases, which can significantly simplify database management.
```

Comment:
I think having the line `Unlike other solutions ...` as its own paragraph makes it stand out a bit more. (In its current state, that line would be combined with the line above it.)

```suggestion
In other words, they virtually unify databases in an incomplete way. For example, with data federation engines, users can run read-only analytical queries on a virtualized view across multiple databases. However, they often need to run update queries separately for each database.

Unlike other solutions, ScalarDB stands out by offering the ability to run both transactional and analytical queries on heterogeneous databases, which can significantly simplify database management.
```